### ISSUE TITLE
Option to not insert blank lines between headings.

### ISSUE THREAD

**Issue by rahulsalvi** [nvim-orgmode/orgmode#58]
### Does this feature exist in Emacs orgmode core?

Yes

### Orgmode link

http://doc.endlessparentheses.com/Var/org-blank-before-new-entry.html

**Comment by mortezadadgar** [comment]
@kristijanhusak is it possible to change the behaviour of orgmode-org_meta_return
your plugin currently insert a line under the header, I want it to be like doom emacs
doom emacs
![image](https://user-images.githubusercontent.com/27911493/130313069-7fc2daac-9065-4fe9-b3f7-83a92ed31df7.png)
nvim
![image](https://user-images.githubusercontent.com/27911493/130313081-f6365a35-1236-407a-aa2f-61ff70eb11e9.png)

**Comment by kristijanhusak** [comment]
@rahulsalvi @mortezadadgar Support for this is added. You can see it in docs [here](https://github.com/nvim-orgmode/orgmode/blob/master/DOCS.md#org_blank_before_new_entry)

TL;DR, add this to your orgmode config:
```lua
  org_blank_before_new_entry = {
    heading = false,
    plain_list_item = false,
  }
```

**Comment by steigerth** [comment]
If I hit `<Leader><CR>` the new TODO headline is not inserted after the very of the current TODO as it can be seen in the following screencast

[![asciicast](https://asciinema.org/a/tptnscrJGJuAJKuAJnYuqHkmu.svg)](https://asciinema.org/a/tptnscrJGJuAJKuAJnYuqHkmu)

Is this the normal behavior?

**Comment by kristijanhusak** [comment]
@steigerth yeah, that's normal. To add at the end, do `<leader>oit`.
